### ISSUE TITLE
Make parent constructor check explicit when extending from null

### ISSUE THREAD

**Issue by stefanpenner** [6to5/6to5#386]
given:

```js
class Bar {}
class Foo extends Bar {}
```

the Foo constructor conditionally apply's `_Bar`

```js
  var _Bar = Bar;
  var Foo = function Foo() {
    if (_Bar) {
      _Bar.apply(this, arguments);
    }
  };
```

I may be mistaken, so please point  me to the part of the spec that allows for this. I believe if `Bar` is missing this should be a failure scenario.

**Comment by sebmck** [comment]
It's to allow for inheriting from `null`.

```javascript
var Bar = null;
class Foo extends Bar {}
```

**Comment by stefanpenner** [comment]
would it instead make sense to special case the `null` case? Or at-least an explicit `if(_Bar !== null) {..}` so that a pesky `undefined` doesn't sneak in?

**Comment by sebmck** [comment]
Doesn't make much difference but it wouldn't hurt to make it more explicit.

**Comment by stefanpenner** [comment]
> Doesn't make much difference

if the superclass becomes undefined do to a mistake, the failure may be entirely masked

**Comment by thejameskyle** [comment]
I'll take this, I'm tired of working on the site.

**Comment by sebmck** [comment]
Released as of 2.6.0
